app/src/main/report/reducers/reportsReducer.ts

```diff
@@ -20,6 +21,8 @@ export const reportsReducer = (state = reportsInitialState, action: ReportsActio
     switch (action.type) {
         case ReportTypeKeys.REPORTS_FETCHED:
             return { ...state, reports: action.data };
+        case ReportTypeKeys.REPORTS_SORTED:
+            return { ...state, reports: clone(state.reports).sort((a: Report, b: Report)=> a[action.data] < b[action.data] ? -1 : 1) };
```

Comment:
If I'm reading this logic correctly, sorting by `latest_version` will result in older versions at the top of the list, whereas we want newer versions at the top. Pulling out the sorting logic into its own method would also make this a little easier to read.